datumaro/plugins/icdar_format/converter.py

```diff
@@ -56,6 +56,7 @@ def write(self, path):
         os.makedirs(path, exist_ok=True)
         for item in self.annotations:
             file = osp.join(path, 'gt_' + item + '.txt')
+            os.makedirs(osp.dirname(file), exist_ok=True)
```

Comment:
```suggestion
            file = osp.join(path, osp.dirname(item),
                'gt_' + osp.basename(item) + '.txt')
            os.makedirs(osp.dirname(file), exist_ok=True)
```

The item can have a relative path.